lib/boolean-values.js

```diff
@@ -20,7 +20,6 @@ const FALSE_VALUES = new Set([
 , 'no'
 , 'N'
 , 'n'
-, ''
```

Comment:
do we actually want to remove this? I think empty string should still evaluate to false for a boolean - I'm assuming this got tangled up in the default values?

Reply:
easier said that done.  Since the empty string triggers special features now, this means something.

I can have another perhaps.  I don't disagree that it's intuitive that `''` means false.  I had taken it out early on cause it broke a bunch of things.

Reply:
ok, I remember now.  This one gets a little tricky.  There's a test that iterates through all of these "false" values and makes sure it passes validation.  However, the definition it uses has `required()`, and by our new rules, `''` is not a valid value for required.

SO--if we want this to be the case only for boolean (that '' is actually a valid `false`), it'll have to have special handling.  OR--we could not do magic and say that if something is `required`, then you have to provide `'false'` 🤷🏻‍♂️

Reply:
nvm, I just fixed the test.  It shouldn't be using `required`

Reply:
ahhh that makes sense, nice 👍